rust/feature-flags/src/flags/flag_request.rs

```diff
@@ -506,6 +508,8 @@ mod tests {
             redis_reader_client.clone(),
             redis_writer_client.clone(),
             pg_client.clone(),
+            432000, // team_cache_ttl_seconds
+            432000, // flags_cache_ttl_seconds
```

Comment:
can we make this a constant, and maybe with a bit more context (I reckon this is 5 days, but would prefer a constant for that).

Reply:
or even just explain in the comment, given that in the app code we'll just use the config value